### ISSUE TITLE
Add plotting extension for Makie-style recipes

### ISSUE THREAD

**Issue by stillyslalom** [JuliaPhysics/ABCDMatrixOptics.jl#7]
I'll try to put together a PR for this later this week.

**Comment by roflmaostc** [comment]
Would be cool! I was wondering if we can also use Makie to draw lenses, etc.

That would be awesome!

**Comment by roflmaostc** [comment]
Would you be still up to make this :)?

**Comment by stillyslalom** [comment]
Yeah, I took a shot later that week but ran into some snarls with Makie's recipe system. I'll give it another try over winter break.

**Comment by roflmaostc** [comment]
Yeah no worries, I don't need that atm
Thanks!

**Comment by roflmaostc** [comment]
> I'll try to put together a PR for this later this week.

Classic GitHub haha

**Comment by stillyslalom** [comment]
Yeah, I use this package for optics layouts every couple of months, and every time I go back to this issue and ask whether I can spare time to try wrangling Makie's recipes, then I go back to using the Plots extension 😅

Upside of being done with grad school is being paid a lot more to work a standard 40 hour week, downside is that it's harder to justify spending work hours on open source, and I've lost almost all motivation to develop code when not being paid to do so. But I owe this issue some of my free time at this point...
